ipm_library/package.xml

```diff
@@ -10,12 +10,8 @@
   <depend>geometry_msgs</depend>
```

Comment:
Missing dep on `builtin_interfaces`.

Reply:
Thanks for picking this up, I've added it in both package.xml files.